### DISCUSSION TITLE
Add consequences for having rioters in poor neighborhoods and no prefect killing them (they threaten to burn down the city but never do)

### DISCUSSION THREAD

**Discussion by thebashsupremacy** [Keriew/augustus#363] [Ideas]
That will add the incentive to maybe have prefectures at least in poorer neighborhoods even in northern maps and will give their threat to burn down the city meaning.

**Comment by crudelios** [comment]
This has been addressed for Augustus v3.0.

Expect a lot more crime if you don't treat your citizens well!
